pkg/startup/v7/derive.go

```diff
@@ -27,17 +32,16 @@ func (derivedType) SystemReserved(cs *api.OpenShiftManagedCluster, role api.Agen
 		if isSmallVM(app.VMSize) {
 			if role == api.AgentPoolProfileRoleMaster {
 				return "cpu=500m,memory=1Gi", nil
-			} else {
-				return "cpu=200m,memory=512Mi", nil
 			}
 
-		} else {
-			if role == api.AgentPoolProfileRoleMaster {
-				return "cpu=1000m,memory=1Gi", nil
-			} else {
-				return "cpu=500m,memory=512Mi", nil
-			}
+			return "cpu=200m,memory=512Mi", nil
```

Comment:
changed the logic here because `go vet` was complaining it wasn't optimal, basically just removed the else statements.